Gemfile

```diff
@@ -56,9 +59,10 @@ group :system_tests do
   gem 'beaker-pe',                                                               :require => false if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0')
   gem 'beaker-rspec', *location_for(ENV['BEAKER_RSPEC_VERSION'] || '>= 3.4')     if ! supports_windows
   gem 'beaker-rspec', *location_for(ENV['BEAKER_RSPEC_VERSION'] || '~> 5.1')     if supports_windows
-  gem 'beaker-puppet_install_helper',                                            :require => false
-  gem 'master_manipulator',                                                      :require => false
+  gem 'beaker-puppet_install_helper',:path => '/Users/wilson/repos/beaker-puppet_install_helper'
+  gem 'master_manipulator', :path => '/Users/wilson/repos/master_manipulator'
```

Comment:
If you put a git repo and branch here, travis can check it out for you and you get better feedback.